ui-admin/src/portal/PortalEnvConfigView.tsx

```diff
@@ -27,6 +28,9 @@ const PortalEnvConfigView = ({ portalContext, portalEnv }: PortalEnvConfigViewPr
   const updateConfig = (propName: string, value: string | boolean) => {
     setConfig(set(propName, value))
   }
+  useUpdateEffect(() => {
```

Comment:
Sorry for my inexperience, but trying to understand how the impact of this is differs, from the `config` initialization on line 23. I assume the initialization on line 23 sets the config for the current render, and this `setConfig` here applies to a the next render (which may happen right away)?
Also, how do the two dependencies relate to the operation within the clause?